great_expectations/datasource/fluent/config.py

```diff
@@ -163,12 +163,12 @@ def update_datasources(self, datasources: Dict[str, Datasource]) -> None:
         datasources_as_dict.update(datasources)
         self.fluent_datasources = list(datasources_as_dict.values())
 
-    def pop(self, datasource_name: str, default: T = _MISSING) -> Datasource | T:  # type: ignore[assignment] # sentinel value is never returned
+    def pop_datasource(self, name: str, default: T = _MISSING) -> Datasource | T:  # type: ignore[assignment] # sentinel value is never returned
```

Comment:
This is to bring this name into alignment with `get_datasource`. Our pattern for methods is `verb_type(name)`.